units/src/result.rs

```diff
@@ -265,6 +267,7 @@ impl fmt::Display for MathOp {
             MathOp::Div => write!(f, "div"),
             MathOp::Rem => write!(f, "rem"),
             MathOp::Neg => write!(f, "neg"),
+            MathOp::Other => write!(f, "other"),
```

Comment:
This is not descriptive enough.

Reply:
Removed and am using Mul instead since it's the wrapped operation.